src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int ballsToCreate = 3;
+        Lottery lottery = new Lottery();
+        for (int i = 1; i <= ballsToCreate; i++) {
+            System.out.println(lottery.getRandomBall());
```

Comment:
not a mistake, but better get used to iterating from 0, and with exit condition of <, not <=, it's the most common way